applications/osb-portal/src/pages/RepositoriesPage/RepositoriesPage.tsx

```diff
@@ -289,7 +289,15 @@ export const RepositoriesPage = ({ user }: { user: UserInfo }) => {
 
         {repositories ?
           <Box className="verticalFill" style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-between' }}>
-            <Repositories repositories={repositories} handleRepositoryClick={(repositoryId: number) => openRepoUrl(repositoryId)} refreshRepositories={() => updateList(tabValue)} />
+            <Repositories
+            repositories={repositories}
+            refreshRepositories={() => updateList(tabValue)}
+            searchFilterValues={searchFilterValues}
+            handleRepositoryClick={(repositoryId: number) => openRepoUrl(repositoryId)}
+            handleTagClick={ (tag: Tag) => searchFilterValues.tags.indexOf(tag.tag) === -1 ? setSearchFilterValues({ ...searchFilterValues, tags: searchFilterValues.tags.concat(tag.tag)}) : console.log("Tag exists, no op.") }
```

Comment:
`searchFilterValues.tags.indexOf(tag.tag) === -1` is equivalent to the more readable `searchFilterValues.tags.find(tag.tag)` ;)

Reply:
Hrm, is that the correct usage of `find`? It seems to require a function as an argument?

I think `includes` is what we want here?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
used `includes` instead of `indexOf`, seems to work fine.